### PR TITLE
Only show metadataType with 2 or more values

### PR DESCRIPTION
**Pull request by haideriqbal**
We don't want to show metadata types in tsne plot dropdowns which only have 1 value.

### CODE REVIEW

File: src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataService.java [hunk 1]
```diff
@@ -28,6 +28,10 @@ public CellMetadataService(CellMetadataDao cellMetadataDao) {
                 characteristicTypes);
     }
 
+    public Map<String, String> getMetadataValuesForGivenType(String experimentAccession, String metedataType) {
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataService.java:31]: **metedataType** spelling correction, I know it's very minor but helpful.

Comment (haideriqbal) [reply src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataService.java:31]: metedataType? sorry I didn't get it :)

File: src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java [hunk 2]
```diff
@@ -144,7 +145,18 @@ public JsonArray getPerplexities(String experimentAccession) {
...
+                .stream()
+                .filter(type -> cellMetadataService
+                        .getMetadataValuesForGivenType(experimentAccession, type)
+                        .values()
+                        .stream()
+                        .distinct()
+                        .count() >= 2)//we only need to show those metadata type which have more than 2 unique values
+                .collect(ImmutableSet.toImmutableSet());
+
+        metadataTypes
```
Comment (lingyun1010) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java:159]: I think we could remove the declaration for the new variable `metadataTypes` by merging the stream calculation.

Comment (haideriqbal) [reply src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java]: did it!

**Pull request merged**